prow/config/config.go

```diff
@@ -925,6 +942,21 @@ func parseProwConfig(c *Config) error {
 		c.PodNamespace = "default"
 	}
 
+	if c.GitHubOptions.LinkURL == "" {
+		c.GitHubOptions.LinkURL = "https://github.com"
+	} else {
+		if _, err := url.Parse(c.GitHubOptions.LinkURL); err != nil {
```

Comment:
nit: This validation doesn't need to be in an `else` block here or below. Moving this out of the `else` block adds the benefit of validating the hardcoded defaults.

Reply:
Makes sense... moved it out.